### PR TITLE
Minor improvements to DC related stuff

### PR DESCRIPTION
**Pull request by varunagrawal**
Isn't it "Marvel"-ous?

### CODE REVIEW

File: gtsam/hybrid/tests/testHybridFactorGraph.cpp [hunk 1]
```diff
@@ -331,13 +331,15 @@ GaussianFactorGraph::shared_ptr batchGFG(double between,
...
 TEST(DCGaussianElimination, Eliminate_fully) {
-  Switching self(2);
+  Switching self(2, 1.0, 0.1);
   auto factors = self.linearizedFactorGraph;
 
   // Add measurement factors
   auto measurement_noise = noiseModel::Isotropic::Sigma(1, 0.1);
-  factors.emplace_gaussian<JacobianFactor>(X(1), I_1x1, 1 * Vector1::Ones(), measurement_noise);
-  factors.emplace_gaussian<JacobianFactor>(X(2), I_1x1, 2 * Vector1::Ones(), measurement_noise);
+  factors.emplace_gaussian<JacobianFactor>(X(1), I_1x1, 1 * Vector1::Ones(),
```
Comment (dellaert) [inline gtsam/hybrid/tests/testHybridFactorGraph.cpp:339]: undo this change, will cause conflict with Fan's branch (measurements added in Switching now)

File: gtsam/linear/HessianFactor.cpp [hunk 2]
```diff
@@ -260,12 +260,12 @@ HessianFactor::HessianFactor(const GaussianFactorGraph& factors,
 void HessianFactor::print(const std::string& s,
     const KeyFormatter& formatter) const {
   cout << s << "\n";
-  cout << " keys: ";
+  cout << " keys: [";
```
Comment (dellaert) [inline gtsam/linear/HessianFactor.cpp:263]: please undo changes in this file. Touching print in core GTSAM is off limits for now

**Review by dellaert — APPROVED** [review]

File: gtsam/linear/HessianFactor.cpp (hunk 2 above)
Comment (varunagrawal) [reply gtsam/linear/HessianFactor.cpp:263]: Gotcha. Will save it for later then.
